native/cocos/renderer/gfx-gles2/GLES2Wrangler.cpp

```diff
@@ -83,3 +114,12 @@ bool gles2wInit() {
 
     return true;
 }
+
+
+bool gles2wExit() {
+    if (!gles2wClose()) {
```

Comment:
What's the difference between gles2wExit & gles2wClose?

Reply:
The exit process mimics the initialization process：gles2wInit->gles2wOpen